config/schedule.rb

```diff
@@ -0,0 +1,3 @@
+every 1.day, at: '4:30 am' do
+  command "cd /srv/frontend/ && source /etc/mas/environment && RAILS_ENV=production bundle exec rake feed:store"
```

Comment:
The `job_type` didn't work?

Reply:
@osharpe mentioned his preferred approach to have the same as cms (which confirmed works). Since I didn't had strong opinions I just follow the approach.

Reply:
ye wasn't too sure about the carriage return, so thought prob best to go with what we know works. We can of course test this `job_type` in test environment and use it if it add some value.